deeppavlov/core/models/keras_model.py

```diff
@@ -17,13 +17,15 @@
 from copy import deepcopy
 from logging import getLogger
 from typing import Optional, List, Union
+from overrides import overrides
 
 import numpy as np
 import tensorflow as tf
 from keras import backend as K
 
 from deeppavlov.core.models.nn_model import NNModel
 from deeppavlov.core.models.tf_backend import TfModelMeta
+from .lr_scheduled_model import LRScheduledModel
```

Comment:
```suggestion
from deeppavlov.core.models.lr_scheduled_model import LRScheduledModel
```
or the other way around for other local imports